pshtt/pshtt.py

```diff
@@ -127,6 +133,14 @@ def result_for(domain):
 
 
 def ping(url, allow_redirects=False, verify=True):
+    # If there is a custom CA file and we want to verify
+    # use that instead when pinging with requests
```

Comment:
That's my bad, I added that comment, it should have a comma after `verify`. The `use` word is correct though. I'll fix it in a later commit.